cmd/postcli/main.go

```diff
@@ -97,6 +109,12 @@ func main() {
 		return
 	}
 
+	if printNumFiles {
+		totalFiles := initialization.TotalFiles(cfg, opts)
```

Comment:
I think it makes sense to have totalfiles as method of `cfg` taking `opts` as parameter.

Reply:
Updated to be a method of `InitOpts`.